components/tools/OmeroPy/test/integration/gatewaytest/test_get_objects.py

```diff
@@ -378,52 +378,45 @@ def testGetDatasetsByProject(self, gatewaywrapper):
 
     def testListExperimentersAndGroups(self, gatewaywrapper):
         gatewaywrapper.loginAsAuthor()
-        # experimenters
-        # experimenters = list( gatewaywrapper.gateway.listExperimenters() ) #
-        # removed from blitz gateway
-        # all experimenters
-        exps = list(gatewaywrapper.gateway.getObjects("Experimenter"))
-
-        # self.assertEqual(len(exps), len(experimenters))  # check unordered
-        # lists are the same length & ids
-        # eIds = [e.getId() for e in experimenters]
+        conn = gatewaywrapper.gateway
+        # experimenters - load_experimentergroups True by default
+        exps = conn.getObjects(
+            "Experimenter", opts={'limit': 10})
         for e in exps:
-            # assert e.getId() in eIds
             # check iQuery has loaded at least one group
-            loaded = False
-            for groupExpMap in e.copyGroupExperimenterMap():
-                if groupExpMap is not None and \
-                        e.id == groupExpMap.child.id.val:
-                    loaded = True
-            assert loaded
-
-        # returns all experimenters except current user - now moved to
-        # webclient_gateway
-        # allBarOne = list( gatewaywrapper.gateway.getExperimenters() )
-        # assert len(allBarOne)+1 ==  len(exps)
-        # for e in allBarOne:
-        #    assert e.getId() in eIds
-
-        # groups
-        # groups = list( gatewaywrapper.gateway.listGroups() )
-        # now removed from blitz gateway.
-        gps = list(gatewaywrapper.gateway.getObjects("ExperimenterGroup"))
+            assert e._obj.groupExperimenterMapLoaded
+            e.copyGroupExperimenterMap()
+
+        # load experimenters without groups
+        opts = {'load_experimentergroups': False, 'limit': 10}
```

Comment:
It will be better to split the test into 2. Loaded true and false.
As it stands if the first check fails, the "loaded: false" is never tested